exchange/exchange-ps/exchange/Set-RetentionCompliancePolicy.md

```diff
@@ -663,9 +676,13 @@ The DeletedResources parameter specifies the deleted Microsoft 365 Group, mailbo
 
 A valid value is a JSON string. Refer to the Examples section for syntax and usage examples of this parameter.
 
-For more information on the deleted Microsoft 365 Group scenario, see [Learn more about modern group deletion under retention hold](https://learn.microsoft.com/purview/retention-settings#what-happens-if-a-microsoft-365-group-is-deleted-after-a-policy-is-applied).
+**CAUTION**: When you use a SMTP address with this parameter, be aware that the same address might also be in use for other mailboxes or mail users. To check for additional mailboxes or mail users with the same SMTP address, use the following command and replace *user@contoso.com* with the SMTP address to check: `Get-Mailbox -RecipientTypeDetails UserMailbox,SharedMailbox | Where-Object { $_.EmailAddresses -contains "SMTP:user@contoso.com" } | Select-Object DisplayName, EmailAddresses, Description, Alias, OrganizationalUnit`
```

Comment:
the cmdlet needs to be changed to (we need to check both mailbox and mailusers which are active or deleted):

Get-Recipient -IncludeSoftDeletedRecipients [user@contoso.com](mailto:user@contoso.com) |Select-Object DisplayName, EmailAddresses, Description, Alias,RecipientTypeDetails, WhenSoftDeleted